javascript/javascript1/week3/Notepad.js

```diff
@@ -0,0 +1,65 @@
+let notes = [];
+
+function saveNote(content, id) {
+  // The function is to save notes like content (message)and id(like priority)
+
+  notes.push({ content, id });
+}
+function getNote(id) {
+  // the function is to find notes in the array of objects
+  if (id == null || isNaN(id)) {
+    return "getNote argument is null or not a number";
+  }
+  for (let i = 0; i < notes.length; i++) {
+    if (notes[i].id === id)
+      return "content is '" + notes[i].content + "' for the id " + notes[i].id;
+  }
+  return "content can not found for id " + id;
+}
+function logOutNotesFormatted() {
+  // your code here
+  for (let i = 0; i < notes.length; i++) {
+    console.log(
+      "The note with id: " +
+        notes[i].id +
+        ", has the following note text: " +
+        notes[i].content
+    );
+  }
+}
+// unique feature I think would make this app better is
+// to save a finished note(remove content and change priorities)
+function endedNote(id) {
+  // the function takes id a parameter as parameter
+  // it removes content and id from the notes array
+  // and it updates notes array by changing ids for the rest of objects
+  if (id == null || isNaN(id) || id > notes.length) {
```

Comment:
I don't recommend using `isNaN` 
remember that it is a good practice to use `===` not `==`